projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/ExecutionTerminationStatusConverter.java

```diff
@@ -9,18 +9,22 @@
 import javax.persistence.Converter;
 import java.util.Arrays;
 
+import com.google.common.primitives.Ints;
+
 @Converter
-public class ExecutionTerminationStatusConverter implements AttributeConverter<ExecutionTerminationStatus, Integer> {
+public class ExecutionTerminationStatusConverter implements AttributeConverter<ExecutionTerminationStatus, String> {
```

Comment:
It is a good idea to have a test that verifies that a bug fix is truly fixed. This way we increase our confidence in the fix, make sure we won't regress again.

Reply:
Done